src/styles/main.css

```diff
@@ -0,0 +1,189 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The hardcoded width of 200px in grid-template-columns could be replaced with a variable for better maintainability and flexibility.